gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@ material = "1.9.0"
 #AndroidX
 activity-compose = "1.7.2"
 appcompat = "1.6.1"
-compose = "1.6.1"
+composeBom = "2024.02.00"
```

Comment:
2024.02.01 is ready now

Reply:
yes, but that is compose 1.6.2, I would rather to the upgrade itself separately than this versioning switching.